pegasus/sites.v3/code.org/views/homepage_hero.haml

```diff
@@ -188,6 +191,13 @@
     showQuote(lastImage, currentImage);
   }
 
+  function clickHero() {
+    heroTarget = $("#fullwidth"+currentImage).attr("data-click-url");
+    if (heroTarget !== null && heroTarget !== undefined) {
+      window.location.href = heroTarget;
+    }
+  }
+
   $.ready()
   {
```

Comment:
sorry, this isn't part of your diff, but I'm really confused about the syntax here. Shouldn't the code block below this line be wrapped inside a function that is given as a parameter to `$.ready()`? As in

```js
$.ready(function () {
  ...
});
```

Otherwise the `$.ready` isn't really doing anything?